Model/Import/Attributes.php

```diff
@@ -428,4 +431,28 @@ private function updateFilterCategoryMapping($sinch_feature_id, $sinch_category_
         $this->filterMappingInsert->execute();
         $this->filterMappingInsert->closeCursor();
     }
+
+    private function clearAllAttributeCode()
```

Comment:
Why is this being added?

Reply:
Attributes that are old or have been removed from the csv file will not be deleted from the eav_attribute table, this will affect when displayed outside the frontend.